plugins/azure-devops-common/src/types.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum BuildResult {
```

Comment:
Do you wanna maybe delete these from the `backend` plugin and start using this common library?

Reply:
Ok that make sense, would be good to get that PR to ready state so we can get it in!

Reply:
Yeah I'm working getting that one published as soon as this one goes through...